system-image-upgrader

```diff
@@ -43,9 +43,6 @@ check_filesystem() {
 verify_signature() {
     # $1 => validation keyring name
     # $2 => path to validate
-  #  if [ -e /etc/system-image/skip-gpg-verification ]; then
```

Comment:
We decided we want to keep the option to skip verification, could you update and just remove the # signs? Thx ^^